Libraries/LibCore/CLocalServer.cpp

```diff
@@ -71,7 +74,19 @@ bool CLocalServer::listen(const String& address)
 
     int rc;
 
+#ifdef SOCK_NONBLOCK
     m_fd = socket(AF_LOCAL, SOCK_STREAM | SOCK_NONBLOCK | SOCK_CLOEXEC, 0);
+#else
+    m_fd = socket(AF_LOCAL, SOCK_STREAM, 0);
+#ifdef _WIN32
```

Comment:
Is there a Windows port in the works too? 😱

Reply:
Why not? 😄

Reply:
Do we even need `CLocalServer` on the host for something?

Reply:
Not directly, but it's part of the base libraries built for host tools (IPCCompiler etc.).
I think it's easier to fix compiling that file on non-Linux hosts rather than adding complexity to makefiles when compiling host tools.

Reply:
I anticipate that Unix-specific parts of LibCore (servers, sockets, pipes and stuff like that — not utilities like `CObject` and `CFile`) are only going to become more Serenity-specific; and there should not be any need for them during code generation on the host, so I would opt for only building them for Serenity.

But then maybe they're *supposed* to be ported as a part of Lagom, so let's defer to @awesomekling's judgement :)

Reply:
Hmm. Please fix LibCore mac stuff in a separate commit if it’s not too invasive. I agree it’s easier to just fix the build instead of fixing IPCCompiler and pals to not use LibCore. :)

Reply:
I have tried splitting the fixes in multiple specific commits, as per your requests.